cdc/sink/mysql/mysql_syncpoint_store.go

```diff
@@ -222,7 +222,7 @@ func (s *mysqlSyncPointStore) SinkSyncPoint(ctx context.Context,
 	// insert ts map
 	query := "insert ignore into " + schemaName + "." + syncPointTableName +
 		"(ticdc_cluster_id, changefeed, primary_ts, secondary_ts) VALUES (?,?,?,?)"
-	_, err = tx.Exec(query, s.clusterID, id.Namespace+"_"+id.ID, checkpointTs, secondaryTs)
+	_, err = tx.Exec(query, s.clusterID, id.ID, checkpointTs, secondaryTs)
```

Comment:
Does the `id.Namespace` same as `s.clusterID`?

Reply:
no

Reply:
`clusterID` stand for the cdc's cluster ID and `Namespace` is the namespace of the user.